compose/compiler/compiler-hosted/integration-tests/src/test/java/androidx/compose/compiler/plugins/kotlin/ControlFlowTransformTests.kt

```diff
@@ -483,29 +520,18 @@ class ControlFlowTransformTests : AbstractControlFlowTransformTests() {
         """
             @NonRestartableComposable @Composable
             fun Example(x: Int?) {
-                // the composable call is made conditionally, which means it is like an if, but one
-                // with static groups, so no wrapping group needed.
+                // The composable call is made conditionally, which means it is like an if with 
+                // only one result having a composable call, so we just generate a single group 
```

Comment:
This line has trailing whitespace, which triggers a lint warning in our CI. Could you please remove it? Sorry, I didn't catch this earlier.